mkosi

```diff
@@ -1953,17 +1942,26 @@ def install_centos(args: CommandLineArguments, root: str, do_run_build_script: b
 
     packages = ['centos-release']
     packages += args.packages or []
-    if args.bootable:
-        packages += ["kernel", "systemd-udev", "binutils"]
     if do_run_build_script:
         packages += args.build_packages or []
+    if args.release == "8":
+        if args.bootable:
+            packages += ["kernel", "systemd-udev", "binutils"]
```

Comment:
Why pull in `systemd-udev` here? Don't we need the entire `systemd` package here as well? And is `binutils` strictly necessary for a bootable image?

Reply:
Actually, now that I look again, it seems the previous PR already added code for the bootable option. Can you check if that works correctly already for both centos 7 and centos 8? If so, these changes can be reverted as well

Reply:
I just tested, the reason I originally separated them out because centOS 7 does not have systemd-udev, so it needs systemd, however I never implemented bootable support for centOS 7, and it currently is not working on this PR. However you can just resolve these lines into what the original PR had, and so I reverted it since it is cleaner.